gcp_variant_transforms/libs/processed_variant.py

```diff
@@ -261,6 +266,67 @@ def _create_annotation_map(self, annotations, annotation_names):
+    """Adds the alternate_bases record to the given schema.
+
+    Depending on how this class is set up to split INFO fields among alternate
+    bases, this function produces a compatible alternate_bases record that is
+    added to the given schema.
+    """
+    alternate_bases_record = bigquery.TableFieldSchema(
+        name=bigquery_util.ColumnKeyConstants.ALTERNATE_BASES,
+        type=bigquery_util.TableFieldConstants.TYPE_RECORD,
+        mode=bigquery_util.TableFieldConstants.MODE_REPEATED,
+        description='One record for each alternate base (if any).')
+    alternate_bases_record.fields.append(bigquery.TableFieldSchema(
+        name=bigquery_util.ColumnKeyConstants.ALTERNATE_BASES_ALT,
+        type=bigquery_util.TableFieldConstants.TYPE_STRING,
+        mode=bigquery_util.TableFieldConstants.MODE_NULLABLE,
+        description='Alternate base.'))
+    if self._split_alternate_allele_info_fields:
+      for key, field in self._header_fields.infos.iteritems():
+        if field.num == vcf.parser.field_counts[_FIELD_COUNT_ALTERNATE_ALLELE]:
+          alternate_bases_record.fields.append(bigquery.TableFieldSchema(
+              name=bigquery_util.get_bigquery_sanitized_field_name(key),
+              type=bigquery_util.get_bigquery_type_from_vcf_type(field.type),
+              mode=bigquery_util.TableFieldConstants.MODE_NULLABLE,
+              description=bigquery_util.get_bigquery_sanitized_field(
+                  field.desc)))
+
+    for annot_field in self._annotation_field_set:
+      if annot_field not in self._header_fields.infos:
+        raise ValueError('Annotation field {} not found'.format(annot_field))
+      annotation_names = _extract_annotation_names(
+          self._header_fields.infos[annot_field].desc)
+      annotation_record = bigquery.TableFieldSchema(
+          name=bigquery_util.get_bigquery_sanitized_field(annot_field),
+          type=bigquery_util.TableFieldConstants.TYPE_RECORD,
+          mode=bigquery_util.TableFieldConstants.MODE_REPEATED,
+          description='List of {} annotations for this alternate.'.format(
+              annot_field))
+      for annotation_name in annotation_names:
+        annotation_record.fields.append(bigquery.TableFieldSchema(
+            name=bigquery_util.get_bigquery_sanitized_field(annotation_name),
+            type=bigquery_util.TableFieldConstants.TYPE_STRING,
+            mode=bigquery_util.TableFieldConstants.MODE_NULLABLE,
+            # TODO(bashir2): Add descriptions of well known annotations, e.g.,
+            # from VEP.
+            description=''))
+      alternate_bases_record.fields.append(annotation_record)
+    schema.fields.append(alternate_bases_record)
+
+  def info_is_in_alt_bases(self, info_field_name):
+    # type: (str) -> bool
+    if info_field_name not in self._header_fields.infos:
+      raise ValueError('INFO field {} not found'.format(info_field_name))
+    if ((self._split_alternate_allele_info_fields and
```

Comment:
nit: you can just do return (...)

Reply:
Done.

Reply:
FYI, I changed this a little because pylint was suggesting to do it like [this](https://travis-ci.org/bashir2/gcp-variant-transforms/builds/344996460) (check the bottom of the page) and it did not seem to be very readable. So instead I do it the way it is now.